events/hfe_events_init.txt

```diff
@@ -873,6 +884,7 @@ country_event = {
 				trait = trait_psionic
 			}
 			immortal = yes
+			species_bio = "The almighty men of Ju Ahet. Descended from the Old Kingdom Ancient Egyptians, who built the pyramids, this race was initially enslaved by the Ahets before rebelling. While Ancient Egypt considered blue to be the holy color, the Ju Ahets consider purple to be so instead, due to ''direct revelations'' from Ra, their chief god.'"
```

Comment:
Should be loc string

Reply:
Species Biographies cannot use localization strings. They must be directly written into the file. Not even Paradox uses them.